config/dev.js

```diff
@@ -48,6 +48,9 @@ module.exports = merge(base, {
 		contentful: {
 			environment: 'development'
 		},
+		intercom: {
+			enable: false,
```

Comment:
let's enable in dev for testing. We can set the experiment setting to control(off) 100 but can turn on for a couple of verification tests with CS.